python/deltalake/table.py

```diff
@@ -473,15 +478,48 @@ def update(
         >>> from deltalake import DeltaTable
         >>> dt = DeltaTable("tmp")
         >>> dt.update(updates = {
-        ...             "deleted": True,
+        ...             "deleted": 'True',
         ...             "id": "concat(id, '_old')"
         ...             }
         ...         )
 
+        Update some row values with python object. This is equivalent to
```

Comment:
```suggestion
        To use Python objects instead of SQL strings, use the `new_values` parameter
        instead of the `updates` parameter. For example, this is equivalent to
```